test.js

```
@@ -0,0 +1,516 @@
+  })
+}
+const db = admin.firestore()
+
+const provider = new JsonRpcProvider(CHAIN_RPC_URL)
+const tokenAbi = [
+  { inputs:[{internalType:'address',name:'account',type:'address'}],
+    name:'balanceOf', outputs:[{internalType:'uint256',name:'',type:'uint256'}],
+    stateMutability:'view', type:'function' }
+];
+
+const FUELSHOP_ADDRESS = process.env.FUELSHOP_ADDRESS;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'FUELSHOP_ADDRESS'. Consider to delete it or rename it.

'FUELSHOP_ADDRESS' is defined here:


(lint/suspicious/noRedeclare)